src/app/feed/page.tsx

```diff
@@ -10,7 +10,7 @@ import { cardSectionContent, wellgorithmContent } from "@/utils/helpers/dummyCon
 
 const Feed: React.FC = () => {
   return (
-    <>
```

Comment:
I prefer the shorthand way of using fragments like before `<>` instead of `React.Fragment`